Source/DafnyLanguageServer/Handlers/Custom/DafnyCounterExampleHandler.cs

```diff
@@ -64,7 +64,7 @@ private class CounterExampleLoader {
           logger.LogDebug("got no counter-examples for document {DocumentUri}", document.Uri);
           return new CounterExampleList();
         }
-        var counterExamples = GetLanguageSpecificModels(document.Counterexamples)
+        var counterExamples = GetLanguageSpecificModels(document.Counterexamples.ToArray())
```

Comment:
?

Reply:
From ConcurrentStack to Array

Reply:
Could you change the signature of `GetLanguageSpecificModels` to take an `IEnumerable`?

Reply:
Removed the ToArray call since `document.Counterexamples` is a List now. `document` comes from DiagnosticObserver.LastPublishedDocument which contains a snapshot of the document that's not mutated, so no concurrency issues should occur here.